src/components/About/Blog/index.js

```diff
@@ -199,29 +198,47 @@ class Blog extends Component {
 
   componentDidMount() {
     const { blogKey } = this.props;
+    this.blogKey = blogKey;
     document.title =
       'Blog Posts about Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots - SUSI.AI';
     if (blogKey) {
-      this.getPosts(blogKey);
+      this.getPosts(0);
     }
   }
 
   componentWillReceiveProps = props => {
     const { blogKey } = props;
+    this.blogKey = blogKey;
     const { posts } = this.state;
     if (posts.length === 0 && blogKey) {
-      this.getPosts(blogKey);
+      this.getPosts(0);
     }
   };
 
-  getPosts = async blogKey => {
-    let payload = await getBlogReponse(blogKey);
+  getPosts = async offset => {
+    this.setState({ postRendered: false });
+    let payload = await getBlogReponse(this.blogKey, 10, offset);
 
     try {
       if (payload.status !== 'ok') {
         throw payload.message;
       }
-      this.setState({ posts: payload.items, postRendered: true });
+      const postsCount = payload.items.length;
+      let nextDisplay = 'visible';
+      let prevDisplay = 'visible';
+      if (postsCount < 10) {
+        nextDisplay = 'hidden';
+      }
+      if (offset === 0) {
+        prevDisplay = 'hidden';
+      }
+      this.setState({
+        posts: payload.items,
+        postRendered: true,
+        startPage: offset,
+        nextDisplay: nextDisplay,
```

Comment:
```
nextDisplay: postsCount < 10 ? 'hidden' : 'visible',
prevDisplay: offset === 0 ? 'hidden' : 'visible',
```

Use this, it will reduce lines of code.